scalaparse/shared/src/main/scala/scalaparse/syntax/Literals.scala

```diff
@@ -32,6 +32,7 @@ trait Literals { l =>
     val ConsumeComments = P( (Basic.WSChars.? ~ Literals.Comment ~ Basic.WSChars.? ~ Basic.Newline).rep )
     P( NoCut( WS ~ Basic.Newline.? ~ ConsumeComments ~ NotNewline) )
   }
+  val TrailingComma: P0 = ("," ~ WS.rep ~ Basic.Newline).?
```

Comment:
@lihaoyi I'm not sure if I should use WS vs WL0 vs WL, cut vs NoCut, etc here. I'm very confused :)